exercises/reverse-string/metadata.yml

```diff
@@ -0,0 +1,5 @@
+---
+blurb: "Given a string, reverse its letters and output a new string."
```

Comment:
This appears to not only be about reversing letters.
We're not outputting anything.

Reply:
How about?
```text
blurb: "Given a string, return a reversed string."
```
Too simple?

Reply:
Not simple enough?

> blurb: "Reverse a string"